test/kt/CMakeLists.txt

```diff
@@ -124,7 +124,49 @@ if (ONEDPL_TEST_ENABLE_KT_ESIMD)
     _generate_esimd_sort_test("256" "32" "double" "" 1000) # segfault
 endif()
 
+function (_generate_scan_test _data_per_work_item _work_group_size _type)
+
+    if ((NOT TARGET build-scan-tests) AND (NOT TARGET run-scan-tests))
+        add_custom_target(build-scan-tests COMMENT "Build all scan kernel template tests")
```

Comment:
Let's rename the umbrella targets to match the KT backend:
 - build-scan-tests -> build-igpu-tests
 - run-scan-tests -> run-igpu-tests

Reply:
Fixed